snarkos/store/rocksdb/tests.rs

```diff
@@ -14,185 +14,170 @@
 // You should have received a copy of the GNU General Public License
 // along with the snarkOS library. If not, see <https://www.gnu.org/licenses/>.
 
-use crate::storage::{rocksdb::RocksDB, DataID, DataMap};
-
-use snarkvm::{
-    compiler::{Map, MapRead},
-    console::{
-        network::{Network, Testnet3},
-        types::Scalar,
-    },
-    utilities::{test_crypto_rng, Uniform},
+use crate::{
+    store::rocksdb::{DataMap, Database, RocksDB},
+    DataID,
 };
+use serial_test::serial;
+use snarkvm::compiler::{Map, MapRead};
 
-// use snarkvm::compiler::{Map, MapRead};
+use std::fs;
 
-pub(crate) fn temp_dir() -> std::path::PathBuf {
-    tempfile::tempdir().expect("Failed to open temporary directory").into_path()
+fn remove_test_dir(network_id: u16) {
+    let _ = fs::remove_dir_all(format!("~/.aleo/storage/.ledger-{}", network_id));
```

Comment:
This will likely interfere with the existing ledger (which i believe lives at `.ledger-3`). We used the temp_dir previously to avoid the confusion.

On a related note, we are likely going to need to modify the `.ledger-{}` storage paths to add support for custom testnets, to solve the same collision issues.